python_tests/test_schedule_ops.py

```diff
@@ -127,7 +127,7 @@ def test_merge_op(self):
         # same merge position -1 and 2 in a 3 dimensional tensor
         # https://github.com/NVIDIA/Fuser/issues/171
         self.check_input_error(
-            lambda fd: fd.sched.merge(fd.t1, -1),
+            lambda fd: fd.sched.merge(fd.t1, -2),
```

Comment:
even though I changed this, I don't think I should.... since they are not causing any codegen failures.